tests/functionaltests/Tests/NSURLSession.mm

```diff
@@ -281,6 +281,7 @@ - (void)URLSession:(NSURLSession*)session downloadTask:(NSURLSessionDownloadTask
 public:
     BEGIN_TEST_CLASS(NSURLSessionTests)
     TEST_CLASS_PROPERTY(L"UAP:AppXManifest", L"NSURL.AppxManifest.xml")
+    TEST_CLASS_PROPERTY(L"ignore", L"true")
```

Comment:
@bbowman This is actually verbatim copypasta'd from another test in this very file 😉